src/blog/2023/10/certified-nodes.md

```diff
@@ -0,0 +1,66 @@
+---
+title: What are Certified Nodes?
+subtitle: Enhanced Security, Quality, and Support 
+date: 2023-10-27
+authors: ["marian-demme"]
+#image: "/blog/2023/10/images/blueprints.png"
+tags:
+    - posts
+    - releases
+    - flowfuse
+---
+We are thrilled to introduce a new feature for our Teams Tier customers - **Certified Nodes for Node-RED**. This new offering is designed to reinforce your flows's robustness by granting you access to Node-RED nodes that stand up to our rigorous quality and security standards.
+
+<!--more-->
+
+## What is a certified node?
+
+A certified node is a module from the Node-RED library that undergoes a certification process, ensuring it adheres to standards that address three core pillars:
+
+**Quality**
+
+- Testing phases for each node.
+- Operational reliability and compatibility.
+
+**Security**
+
+- Proactive resolution of potential vulnerabilities.
+- Revocation of certification for nodes falling short on security, with prompt notifications to affected customers.
+
+**Support**
+
+- Ambitious aim towards effective issue resolution.
+- Assistance for troubleshooting.
+
+## How to use certfied nodes?
+
+Accessing certified nodes is straightforward—they're integrated directly within your Node-RED palette manager, simplifying selection and implementation. All new instances since October 26, 2023, have automatic access to the catalogue. If you want to add Certified Nodes to one of your existing instances, just [contact us](https://flowfuse.com/support/).
```

Comment:
```suggestion
Accessing certified nodes is straightforward—they're integrated directly within your Node-RED palette manager, simplifying selection and implementation. All new instances since October 26, 2023, have automatic access to the catalogue. If you want to add Certified Nodes to one of your existing instances, just [contact us](/support/).
```

Reply:
Should other customers update their stack? Does that work?

Reply:
I attempted a basic restart today and updated the stack, but it didn't pick up the new catalogue. Every new instance is quite straightforward. Considering that it's solely about Teams and Enterprise, I believe it is acceptable to use our support channel for old ones.